ambari-server/src/main/java/org/apache/ambari/server/actionmanager/ActionScheduler.java

```diff
@@ -113,9 +112,6 @@
   @Inject
   private UnitOfWork unitOfWork;
 
-  @Inject
-  private ActionQueue actionQueue;
```

Comment:
Removing `ActionQueue` seems to be outside the scope of this PR.  Was this intended?

Reply:
This PR was mostly to fix the unit tests broken due to the big changes that were undertaken as a part of https://issues.apache.org/jira/browse/AMBARI-20562.

We are moving to an event-based and thereby more ad-hoc model for handling the status and commands between server and agent.

Reply:
@rlevas yes, it is not used anymore in branch codebase so I decided to remove it here

Reply:
@swagle , @mpapirkovskyy , was enabling Kerberos tested using this new model?

Reply:
@rlevas yes, it works.
Some of RU cases still fail but I'm working on fixing it.